### DISCUSSION TITLE
Support for TIFF file format

### DISCUSSION THREAD

**Discussion by kwoeste11** [CropperBlazor/Cropper.Blazor#320] [General]
I don't find anything explicitly stating the cropper does or doesn't support TIFF, but when I try to load a TIFF file via the demo app I get a not found.   It also doesn't load in the app I created when trying to load a TIFF file.    

Is there support for TIFF files in the cropper ?

**Comment by MaxymGorn** [comment]
Hi @kwoeste11. I think following conversation is answer for your question: https://github.com/fengyuanchen/cropperjs/issues/622. I suppose you need convert to more standard format your tiff image for cropper component due to few browsers (only Safari) support tiff format in html img src at least

**Comment by kwoeste11** [comment]
Thanks for the reply.  Kind of what I was expecting.  I thought I would ask though.  I will look at loading and converting them to jpegs.

**Reply by MaxymGorn** [reply]
So this is a big problem of supporting tiff pictures in the browser - it's a pain! Just see compatibility with img format in browsers: https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
<img width="749" alt="image" src="https://github.com/CropperBlazor/Cropper.Blazor/assets/50423072/a35ef17e-75bd-4239-8d92-9f552272d7ef">


In additional, perhaps from your needs you will need png format.

**Reply by kwoeste11** [reply]
Yeah.  Tiffs are a bit different in that they can be multi page too.  It used to be converting between image types in .net and then displaying wasn't an issue, but with .net Core and Blazor, the converting to and from an IBrowserFile / IFormFile to the images has given me some headaches.  I will do some research and see what I can come up with

**Reply by MaxymGorn** [reply]
I recommend using the [SkiaSharp](https://github.com/mono/SkiaSharp) library to convert images to the appropriate format. But this is not the best option, since this library takes up some space. As far as I know, this library has powerful functions for manipulation with images. Of course, you can use another library that suits you better.
